Library/Homebrew/cmd/update-report.rb

```diff
@@ -249,8 +249,21 @@ def updated?
   def migrate_tap_migration
     report[:D].each do |full_name|
       name = full_name.split("/").last
-      next unless (dir = HOMEBREW_CELLAR/name).exist? # skip if formula is not installed.
       next unless new_tap_name = tap.tap_migrations[name] # skip if formula is not in tap_migrations list.
+      if tap == "caskroom/cask"
+        next unless (HOMEBREW_REPOSITORY/"Caskroom"/name).exist?
+        new_tap = Tap.fetch(new_tap_name)
+        new_tap.install unless new_tap.installed?
+        system HOMEBREW_BREW_FILE, "install", name
+        system HOMEBREW_BREW_FILE, "link", name, "--force"
```

Comment:
For now casks are only migrated to taps within Homebrew organisation and I don't think that there are some duplicated or ambiguous formulas here. But it will definitely be no harm if I change `name` to `full_name`, I guess.